src/Uno.UWP/Globalization/ApplicationLanguages.iOS.cs

```diff
@@ -1,11 +1,21 @@
 ﻿using System.Collections.Generic;
 using System.Linq;
-using Foundation;
 
 namespace Windows.Globalization
 {
 	public static partial class ApplicationLanguages
 	{
-		public static IReadOnlyList<string> ManifestLanguages { get; } = NSBundle.MainBundle.PreferredLocalizations.Concat(NSBundle.MainBundle.Localizations).Distinct().ToArray();
+		public static IReadOnlyList<string> ManifestLanguages { get; } = GetManifestLanguages();
+
+		private static string[] GetManifestLanguages()
+		{
+			var manifestLanguages = global::Foundation.NSLocale.PreferredLanguages
```

Comment:
Why shouldn't we use `CurrentUICulture` (maybe for all platforms actually)

Reply:
Depending on .NET Versions, it may not be set properly.